packages/dht/src/dht/routing/Router.ts

```diff
@@ -188,6 +191,19 @@ export class Router {
         this.ongoingRoutingSessions.delete(sessionId)
     }
 
+    onNodeConnected(peerDescriptor: PeerDescriptor): void {
```

Comment:
Should this be called handleNodeConnected() as this is not an event listener but a public function called from outside (don't know if this convention is still in effect?)

Reply:
We currently apply this `onFooBar` naming style in many components, e.g. in `PeerManager#onContactConnected,` `RoutingSession#onRequestSucceeded`, `DiscoverySession#onClosestPeersRequestFailed.` 

The point is that the method is called when some event/action happens, so from the component's point of view it is just an event listener. In many cases the listener is triggered by external component (as that component knows that the event/action happened), but that doesn't make big difference.

Reply:
Maybe let's try to keep the `onXXX` convention (or alternatively use a method name that describe what is done)?